addon/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Test Pilot",
   "name": "testpilot-addon",
-  "version": "0.8.3",
+  "version": "0.8.5",
```

Comment:
Kind of arbitrary... I think we made a few changes to the add-on this sprint, but never got around to bumping the version. We should probably just establish a habit of bumping the add-on version every sprint end